plugins/scaleflux/sfx-nvme.c

```diff
@@ -105,6 +106,11 @@ struct nvme_additional_smart_log {
 	struct nvme_additional_smart_log_item	 erase_timeout_cnt;
 	struct nvme_additional_smart_log_item	 read_timeout_cnt;
 	struct nvme_additional_smart_log_item	 read_ecc_cnt;//retry cnt
+	struct nvme_additional_smart_log_item    non_media_crc_err_cnt;
+	struct nvme_additional_smart_log_item    compression_path_err_cnt;
+	struct nvme_additional_smart_log_item    out_of_space_flag;
+	struct nvme_additional_smart_log_item    physical_usage_ratio;
+	struct nvme_additional_smart_log_item    grown_bb; //grown bad block
```

Comment:
Where are these additional values used? Shouldn't you have a patch for printing them out or something?

Reply:
Good Eye. Yes, we missed the output for MR. Will update soon.

Reply:
shown part was updated.